main.py

```diff
@@ -1,10 +1,41 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
```

Comment:
The context manager is used correctly to open and read the JSON file. However, according to the checklist, you should not overload the context manager by performing operations other than reading data within it. Consider moving the processing of `players` outside the `with` block .